integration/spark2/src/main/scala/org/apache/spark/sql/execution/command/preaaggregate/PreAggregateListeners.scala

```diff
@@ -50,6 +50,12 @@ object LoadPostAggregateListener extends OperationEventListener {
         val childTableName = dataMapSchema.getRelationIdentifier.getTableName
         val childDatabaseName = dataMapSchema.getRelationIdentifier.getDatabaseName
         val selectQuery = dataMapSchema.getProperties.get("CHILD_SELECT QUERY")
+        // REVIEW_COMMENT:
+        // 1. change to use LoadTableCommand runnable command
+        // 2. there maybe concurrent load, for example, datamap load will triggered
+        // after create datamap on main table, and user load main table again immediately, then
+        // datamap load will be triggered again. Will this cause problem? Is there dictionary in
```

Comment:
Data map will use main table dictionary ...it will not have any dictionary file ..

Reply:
So it will work fine..